JavaScript/Node/best-practices/project-structure-continued.md

```diff
@@ -0,0 +1,91 @@
+# Node Best Practices: Project Structure (Cont.)
+author: milesflo
+
+levels:
+
+  - basic
+
+  - advanced
+
+type: normal
+
+category: best practice
+
+standard: js.identify-common-design-patterns
+
+links:
+
+  - >-
+    [Project Structure Practices](https://github.com/i0natan/nodebestpractices#1-project-structure-practices)
+
+---
+## Content
+
+On top of abiding by Separation of Concerns, there are some additional NodeJS-specific best practices that we should address. These are covered with [Express](http://expressjs.com/) in mind, but you can substitute it with any similar HTTP server package.
```

Comment:
Remove this paragraph and then below, change in general to "with regards to [Express](http://expressjs.com/), we should..."